### DISCUSSION TITLE
KPN changed default PPPoE username / password

### DISCUSSION THREAD

**Discussion by DennisR73** [fabianishere/udm-iptv#141] [General]
Hi Fabian,

First of all thanks for putting this software out here! Great work! It has been running successfully for a week now on my UDM PRO SE connected to KPN fiber.

In my case the default PPPoE username/password wasn't working. I was able to retrieve my credentials from the configuration of the KPN Experia box. Looks like they added a MAC address  to the username:  **< MAC Address >@internet** / password: **ppp**.

Just an FYI for other users trying to connect to KPN.

Best,
Dennis

**Comment by fabianishere** [comment]
Interesting. KPN in their documentation says that the username and password do not matter. See https://kpn.com/w3/file?uuid=563993a1-e48a-485a-90a8-7389c5d6f5d4&owner=8de86c61-bdc8-4848-872e-e5b380e682fc&contentid=50183&mode=incontext for fiber.

**Reply by Bolten88** [reply]
I’m planning to install this on my udm se. Now it is running behind the router of KPN. How did you figure out what the username should be? Is it the MAC address of the udm or did you use the MAC address of the KPN router?

**Reply by HammerDownT680** [reply]
From what I read it doesn't matter. However I do see some documentation that says to use MAC address @ internet. For example "aa-bb-cc-dd-ee-ff@internet" and any password. 

And it's of the KPN device

Use google translate on this link:

https://forum.kpn.com/thuisnetwerk-72/gebruik-een-eigen-router-i-p-v-de-experia-box-458609

**Reply by Bolten88** [reply]
Thanks will try that.

what should i configure on this setting?

IPTV_WAN_RANGES

the wan address KPN handed out to me? Or should I leave it default?

**Reply by HammerDownT680** [reply]
There is a config built in for this provider when you run the setup but I believe:

217.166.0.0/16,213.75.0.0/16

Oh also found the password for the username I gave earlier and it's kpn

**Reply by Bolten88** [reply]
Today i rewired the home network. Unplugged the KPN router and the UDM is now directly connected to the NTU. I had to use the mac address login as well to make that work. Installed the iptv package and is working like a charm. I was not aware that there was a small wizard for the configuration. 

Thanks @HammerDownT680 and @fabianishere

**Reply by DennisR73** [reply]
> 

I logged into my Experia box to retrieve the credentials

**Reply by HammerDownT680** [reply]
@Bolten88 No problem!

**Reply by Lennart95** [reply]
I experienced troubles as well. With internet/internet my Dream Router loses connection every 2 minutes. So I want to try your login above. Is the mac-adress in mac-adress@internet the mac-address of the KPN box 12 or can it be the mac-adress of my Dream Router?

**Reply by Bolten88** [reply]
> I experienced troubles as well. With internet/internet my Dream Router loses connection every 2 minutes. So I want to try your login above. Is the mac-adress in mac-adress@internet the mac-address of the KPN box 12 or can it be the mac-adress of my Dream Router?

It should be the MAC address of the KPN router. You can retrieve the correct login name if you login to the KPN router.
